doc/decisions/0007-masking-sensitive-data.md

```diff
@@ -19,7 +19,7 @@ printed out by mistake in a log entry (see https://github.com/status-im/status-m
 
 To minimize the risk of leaking passwords through logs, we should not pass
 passwords as strings in our codebase. We introduced a new type `MaskedData` in
-`status-im.utils.security`.
+`utils.security.core`.
```

Comment:
i don't think we should change outdated files

Reply:
probably better to rollback this change

Reply:
hmmm.. do you think its worth adding a comment over there in the md file stating `status-im.utils.security` is now `utils.security.core` ?

Reply:
my point here being no one should be confused when they find `status-im.utils.security` anywhere in codebase..

Reply:
ok lets keep it, thanks